src/stores/controller.ts

```diff
@@ -162,13 +166,32 @@ export const useControllerStore = defineStore('controller', () => {
 
     if (value === 'hidden') {
       console.warn('Window/tab hidden. Disabling joystick forwarding.')
+      if (vehicleStore.isArmed) {
+        alertStore.pushAlert(
+          new Alert(
+            AlertLevel.Warning,
+            'Warning: Vehicle armed and Cockpit minimized. Joystick inputs are stopped. Ensure vehicle safety.'
+          )
+        )
+        showElectronDesktopNotification(
+          'Vehicle armed and Cockpit minimized. Joystick inputs are stopped. Ensure vehicle safety.'
+        )
```

Comment:
Messages are being replicated here. You can assign them to a variable and reuse. Will also consume less lines.

Also, could we make the message more explicit? Like "Cockpit minimized while vehicle is armed. Joystick inputs will not work. Ensure vehicle safety."?

Reply:
Critical alerts now act independently from the other 4 categories. They are not blocked from the "Enable voice alerts option" and have a special config item to be set on the Configurations->Alerts

![image](https://github.com/user-attachments/assets/78444aeb-eb1d-4f07-95dc-6b29794fbd09)